src/effects/effectslot.h

```diff
@@ -140,6 +138,8 @@ class EffectSlot : public QObject {
     EffectsManager* m_pEffectsManager;
     EffectManifestPointer m_pManifest;
     EffectInstantiatorPointer m_pInstantiator;
+    QSet<ChannelHandleAndGroup> m_pActiveChannels;
```

Comment:
Keep this state only in EffectChainSlot. Duplicating it here introduces the possibility that it will get out of sync with the state of EffectChainSlot in the future. Actually, this already happens when the active input channels are changed after an effect is loaded. You can verify that by:
1. Loading an effect
2. Enabling an input channel
3. Changing a checkbox in DlgPrefLV2
4. Pressing Apply. This crashes Mixxx.